src/layouts/pt-br/categorias.astro

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Correct me if I'm wrong, but the plan would be to remove this and just have one language-agnostic layout, right?